docker/docker-compose.apps.yml

```diff
@@ -30,6 +30,32 @@ services:
       - type: bind
         source: ../apps/proposer/src
         target: /app/src
+        
```

Comment:
Please, add a test for this in Github actions, under the `test-apps` pipeline. I believe that changes should be done in `./bin/setup-nightfall` & `./bin/start-apps` too.